### PR TITLE
Edit button variant and add product card

### PR DESCRIPTION
**Pull request by Maruf-Ahmed76**
I have added a new variant to the button. Also, I have done the product cards

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "singleQuote": true,
-  "tabWidth": 2,
-  "printWidth": 120
+    "singleQuote": true,
+    "tabWidth": 4,
```
Comment (raikusy) [inline .prettierrc:3]:
```suggestion
    "tabWidth": 2,
```

**Review by raikusy — CHANGES_REQUESTED** [review]
Please use tab width 2 spaces. Push another update.

File: src/components/Product Card/ProductCardTop.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                        </Button>
+                    ) : (
+                        <div className="couter_box flex items-center bg-green-500 rounded">
+                            <Button className="p-2" click={() => increment(counter)}>
+                                <BiPlus />
+                            </Button>
+                            <span className="font-sm text-white mx-1">{counter}</span>
+                            <Button className="p-2" click={() => decrement(counter)}>
+                                <BiMinus />
+                            </Button>
```
Comment (shahariar50) [inline src/components/Product Card/ProductCardTop.js:37]:
```suggestion
                            <Button className="p-2" click={() => increment(counter)}>
                               <BiMinus />
                            </Button>
                            <span className="font-sm text-white mx-1">{counter}</span>
                            <Button className="p-2" click={() => decrement(counter)}>
                                 <BiPlus />
                            </Button>
```

File: .prettierrc (hunk 1 above)
Comment (Maruf-Ahmed76) [reply .prettierrc:3]: Alright

File: src/components/Product Card/ProductCardTop.js (hunk 2 above)
Comment (Maruf-Ahmed76) [reply src/components/Product Card/ProductCardTop.js:37]: Need to change the counter function too 🙋🏻‍♂️ Ok got it

**Review by raikusy — APPROVED** [review]
Good work.

**Pull request merged**